tap/tlstapper/tls_poller.go

```diff
@@ -279,27 +273,3 @@ func (p *tlsPoller) logTls(chunk *tlsChunk, ip net.IP, port uint16) {
 		srcIp, srcPort, dstIp, dstPort,
 		chunk.Recorded, chunk.Len, chunk.Start, str, hex.EncodeToString(chunk.Data[0:chunk.Recorded]))
 }
-
-func (p *tlsPoller) SetProtocol(protocol *api.Protocol) {
-	// TODO: Implement
```

Comment:
The existence and implementation of this method might be useful in the future once this logic is removed;

https://github.com/up9inc/mizu/blob/9bddfc19376684a66d0dbea5e21207bfe70d9a23/tap/passive_tapper.go#L74-L77

Reply:
Correct, I just moved it to another place, because tlsPoller is global and not per stream